src/lib/app.ts

```diff
@@ -23,6 +23,23 @@ import ossAuthentication from './middleware/oss-authentication';
 import noAuthentication from './middleware/no-authentication';
 import secureHeaders from './middleware/secure-headers';
 
+function handleCustomAuth({ customAuthHandler, app, config, services }) {
+    const logger = config.getLogger('src/lib/app/customAuthHandler');
+    if (customAuthHandler) {
+        customAuthHandler(app, config, services);
+    } else {
+        app.use(`${config.server.baseUriPath}/api`, async (req, res) => {
+            logger.error(
+                'You have to configure a custom authentication middleware. Read the docs....',
```

Comment:
1. I think we should include a link to https://docs.getunleash.io/docs/deploy/configuring_unleash. 
2. The message should be a constant we reuse in the json response.

Reply:
Agreed. I'll fix it :)